packages/web/src/components/admin_page/url_shortener/table.jsx

```diff
@@ -112,7 +120,13 @@ const UrlsTable = () => {
 
     return (
         <>
-            <OverlayTrigger show={showAddOverlay} placement="bottom" overlay={popover(onUpdate, errorMessage)}>
+            <OverlayTrigger
+                show={showAddOverlay}
+                placement="bottom"
+                overlay={popover(
+                    data => onUpdate(data, triggerFetch, errorMessage, setErrorMessage, setShowAddOverlay),
```

Comment:
Nice. I like this, but some people might have problems reading this. 

I suggest that you add a comment on top of explaining how this anonymous function works.